lib/agenda/index.ts

```diff
@@ -55,6 +56,7 @@ class Agenda extends EventEmitter {
   _findAndLockNextJob: any;
   _indices: any;
   _isLockingOnTheFly: boolean;
+  _isJobQueueFilling: Map<string, boolean>;
```

Comment:
Hey, Leonard.
I know it's outside of the scope of the test you're fixing. But I think your changes is the great time to start a good practice of commenting every property in this class. :) Because currently literally none knows what are all these properties. :)

Could you please add a comment explaining what the property is used for? **Why** it exists? I reckon you can easily add it to both `_isLockingOnTheFly` and `_isJobQueueFilling`.